userspace/libscap/scap.c

```diff
@@ -1169,13 +1169,21 @@ int32_t scap_get_stats(scap_t* handle, OUT scap_stats* stats)
 	return SCAP_SUCCESS;
 }
 
-int scap_get_modifies_state_ppm_sc(uint32_t* ppm_sc_array)
+int scap_get_modifies_state_ppm_sc(OUT uint32_t ppm_sc_array[PPM_SC_MAX])
```

Comment:
Given that C functions really take a pointer, i'd leave a pointer here,
(and same for the tracepoint one).

Reply:
https://github.com/falcosecurity/libs/pull/609#discussion_r971607608 ?